go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/markbates/goth
+module github.com/nik0811/goth
```

Comment:
Thanks for PR, are you able to revert this change?

Reply:
@techknowlogick  yes reverted,